troubleshooting/AIK8sTroubleShooting/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM microsoft/dotnet:2.1-sdk AS build-env
+WORKDIR /app
+
+# Copy csproj and restore as distinct layers
+COPY *.csproj ./
+RUN dotnet restore
+
+# Copy everything else and build
+COPY . ./
+RUN dotnet publish -c Release -o out
```

Comment:
Why not debug?

Just curious--it is for troubleshooting after all :)

Reply:
I will keep it as is for now but I am totally open to change it to debug if necessary.
There is currently almost no user code for debugging per say. That might change later.